pkg/visor/peering_daemon.go

```diff
@@ -0,0 +1,109 @@
+package visor
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net"
+	"net/rpc"
+	"os"
+	"os/exec"
+	"sync"
+	"time"
+
+	"github.com/SkycoinProject/dmsg/cipher"
+	skycoin_cipher "github.com/SkycoinProject/skycoin/src/cipher"
```

Comment:
I think we should only import `"github.com/SkycoinProject/dmsg/cipher"` here. If there are missing functions that we need, we should add it in `"github.com/SkycoinProject/dmsg/cipher"`.

Reply:
@Kifen What is the progress with this?

Reply:
There are functions available in `SkycoinProject/skycoin/src/cipher/crypto.go` which are missing in `github.com/SkycoinProject/dmsg/cipher`. An example is the `MustPubKeyFromHex`, which I made use of in `visor/peering_daemon.go`.